city_scrapers/spiders/chi_northern_il_university.py

```diff
@@ -0,0 +1,134 @@
+from city_scrapers_core.constants import BOARD, COMMITTEE, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+import scrapy
+import datetime
+import re
+
+month_mapping = {
```

Comment:
Generally it's easier to use [`datetime.strptime`](https://docs.python.org/3/library/datetime.html#strftime-strptime-behavior) to parse date strings, so we should go that route here as well